scheduler/scheduler-node/src/main/java/org/ow2/proactive/scheduler/task/executors/ForkedProcessBuilderCreator.java

```diff
@@ -126,4 +129,19 @@ private OSProcessBuilder getOsProcessBuilder(TaskContext context, File workingDi
         }
         return processBuilder;
     }
+
+    private void addExecutionPermissionToScripts() throws IOException {
+        String schedulerHome = ClasspathUtils.findSchedulerHome();
+        //Add execution permissions to scripts except for windows OS
+        if (!StandardSystemProperty.OS_NAME.value().toLowerCase().contains("windows")) {
+            File scriptsDirectory = new File(schedulerHome, "dist/scripts/processbuilder/linux");
+            if (scriptsDirectory != null) {
+                for (File script : scriptsDirectory.listFiles()) {
+                    script.setExecutable(true, false);
```

Comment:
Please, change this instruction to add execution rights to all users:
script.setExecutable(true, false);

Reply:
it is the case with script.setExecutable(true, false);
The second parameter is ownerOnly